scripts/install

```diff
@@ -64,22 +69,63 @@ ynh_app_setting_set "$app" port "$port"
 # Open port in firewall
 yunohost firewall allow Both "$port"
 
+if [ "$mumbleweb" -eq 1 ] ; then
+    port_web=$(ynh_find_port 64800)
+    ynh_app_setting_set "$app" port_web "$port_web"
+    # Open port in firewall
+    yunohost firewall allow Both "$port_web"
+fi
+
 #=================================================
 # INSTALL DEPENDENCIES
 #=================================================
 
 # Install Mumble Debian package via apt
-ynh_install_app_dependencies mumble-server mailutils
+ynh_install_app_dependencies $pkg_dependencies
+
+if [ "$mumbleweb" -eq 1 ] ; then
+    ynh_install_nodejs --nodejs_version="10"
+fi
```

Comment:
If I am not wrong, nodejs is only useful here for building mumble-web. Could it be interesting of rather adding this instruction before the `ynh_use_nodejs` instruction (line 145) and adding the ` ynh_remove_nodejs` instruction after the sub-shell (thus this instruction wouldn't be required in the `remove` script)